docs/telemetry.md

```diff
@@ -30,6 +30,17 @@ The event ping contains a list of events ([see event format on readthedocs.io](h
 | Start session (App is in the foreground) | action   | foreground | app    |        |
 | Stop session (App is in the background)  | action   | background | app    |        |
 
+N.B. Sessions as recorded today are unlikely to match your intuitive definition of a "session". Sessions will sometimes stop when the browser part of the application is partially obscured (`onPause/onResume`). On Echo Show, the session ends for:
+- Opening Firefox settings
```

Comment:
Is this a comprehensive list? Or is this more for "situations you might not expect sessions to stop", in which case, I'd mention that.

Or maybe like "Aside from intuitive situations when the session stops (such as device sleep or switching apps), on the Echo Show the session will also end for:"

Reply:
Good point, done the latter.